src/components/Swap/Speedbump/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { Trans } from '@lingui/macro'
+import Button, { TextButton } from 'components/Button'
+import Column from 'components/Column'
+import { OnCloseDialogContext } from 'components/Dialog'
+import Row from 'components/Row'
+import { PriceImpact } from 'hooks/usePriceImpact'
+import { LargeAlert, LargeIcon, X } from 'icons'
+import { PropsWithChildren, useContext } from 'react'
+import styled, { css } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const SpeedbumpWrapper = styled.div`
```

Comment:
should `Speedbump`  be two words ? like `SpeedBump` ?

Reply:
also, should/can this be a `styled(Column)` ? i've found `Column`  to have pretty sensible defaults

Reply:
good point, will update to Speedbump.

As for column, I was having some trouble w/ some of the defaults there but I'll revisit.